### PR TITLE
Moved autoconfig into config and renamed it to autogradeConfig

### PR DESCRIPTION
**Pull request by icanberk**

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
-require 'autoConfig'
-
 class JobsController < ApplicationController
 
+  autolabRequire File.join(Rails.root, 'config/autogradeConfig.rb')
```
Comment (dlbucci) [inline app/controllers/jobs_controller.rb]: This seems to miss the point of using join. I'd use 'Rails.root.join("config", "autogradeConfig.rb")'

**Comment by icanberk** [comment]
@dlbucci not sure if Rails.root.join is a thing. We seem to use `File.join(Rails.root, ...` throughout the codebase so I changed it to that.

**Comment by dlbucci** [comment]
It absolutely is a thing, I've used it in many places.
See: https://github.com/autolab/Autolab/blob/develop/app/controllers/assessment/autograde.rb#L29

**Comment by icanberk** [comment]
Just for you, Dan.

**Comment by icanberk** [comment]
@mihirpandya you should be aware of this change.

**Pull request merged**